x-pack/legacy/plugins/maps/public/meta.test.js

```diff
@@ -53,20 +47,10 @@ jest.mock('./kibana_services', () => {
 
 describe('default use without proxy', () => {
 
-  it('should return absolute urls', async () => {
-
-    const emsClient = getEMSClient();
-    const tmsServices = await emsClient.getTMSServices();
-
-    const rasterUrl = await tmsServices[0].getUrlTemplate();
-    expect(rasterUrl.startsWith('https://raster-style.foobar')).toBe(true);
-
-    const fileLayers = await emsClient.getFileLayers();
-    const file1Url = fileLayers[0].getDefaultFormatUrl();
-
-    expect(file1Url.startsWith('https://vector-staging.maps.elastic.co/files')).toBe(true);
+  it('should construct EMSClient with absolute manifest url', async () => {
+    getEMSClient();
+    const mockEmsClientCall = EMSClient.mock.calls[0];
+    expect(mockEmsClientCall[0].manifestServiceUrl.startsWith('https://ems-manifest')).toBe(true);
```

Comment:
clever. I learned something new today. Yes, this  essentially tests the same thing, that the EMS-client is instantiated correctly with _not_ using proxying with the default config.